tina/schema.tsx

```diff
@@ -124,20 +124,113 @@ export const schema = defineSchema({
           },
         },
         {
-          name: 'prev',
-          label: 'Prev',
-          type: 'string',
+          name: 'Next',
+          label: 'Next',
+          type: 'reference',
+          collections: ['doc', 'examples'],
         },
         {
-          name: 'next',
-          label: 'Next',
-          type: 'string',
+          name: 'previous',
+          label: 'Previous',
+          type: 'reference',
+          collections: ['doc', 'examples'],
         },
         {
           type: 'rich-text',
           name: 'body',
           label: 'Body',
           isBody: true,
+          templates: [
+            {
+              name: 'Youtube',
+              label: 'Youtube Embed',
+              fields: [
+                {
+                  type: 'string',
+                  name: 'embedSrc',
+                  label: 'Embed URL',
+                  description:
+                    'Looks like this https://www.youtube.com/embed/Yoh2c5RUTiY',
+                },
```

Comment:
I am not sure if this works with other styles of youtube URLs 
e.g. `https://www.youtube.com/watch?v=l0e9i8zXcIs`

The description should be clearer on what input the code can handle
```suggestion
                {
                  type: 'string',
                  name: 'embedSrc',
                  label: 'Embed URL',
                  description:
                    '⚠️ Only youtube embed urls work - they look like this https://www.youtube.com/embed/Yoh2c5RUTiY',
                },
```

Reply:
✅